WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/CategorySectionTableViewCell.swift

```diff
@@ -22,14 +22,15 @@ protocol CategorySection {
     var description: String? { get }
     var thumbnails: [Thumbnail] { get }
     var scrollOffset: CGPoint { get set }
+    var thumbnailSize: CGSize { get }
```

Comment:
Added so `CategorySections` can provide the size of their thumbnails (which all share the same size).